.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,39 +36,88 @@ jobs:
+      #       exit 0
+      #     fi
+
+      #     echo "Previous successful job found: ${SUCCESSFUL_JOB} - skipping the rest of the job"
+      #     echo "SKIP_TESTS=true" >> $GITHUB_ENV
+
+      # Electron requires unprivileged user namespaces to function properly. 
+      # Disabling this security rule allows Electron to create sandboxed processes
+      # without requiring elevated privileges, which is essential for running the application.
+      # This is workaround until electron builder solves this issue in future release.
       - name: Disable security rule 'Restricted unprivileged user namespaces'
+        if: steps.check-previous-runs.outputs.skip-tests != 'true'
```

actionlint (1.7.4)

property "skip-tests" is not defined in object type {skip_tests: string}

(expression)